street_lights/index.html

```diff
@@ -1,12 +1,62 @@
-<!DOCTYPE html>
 <html>
   <head>
     <meta charset="utf-8">
     <title>Światła drogowe</title>
   </head>
   <body>
-    <div id="streetLights">
-      <!-- put here the code for you streetLights -->
+  
+    <div class="streetLights">
+      <div class="light red">
+      </div>
+      <div class="light yellow">
+      </div>
+      <div class="light green">
+      </div>
     </div>
+   
+    <div class="streetLights">
+        <div class="light red off">
```

Comment:
Super pomysł z tym `off` tylko że to trochę takie oszukiwanie bo ukrywasz coś co gdzieś tam jest a nie do końca o to chodzi ten element po prostu nie ma mieć backgrounda jak swiatlo jest wyłączone. 
Czyli inaczej mówiąc ma mieć background tylko wtedy gdy się świeci. 
Przyjmijmy że jak swiatlo się świeci to ma jeszcze dodatkowo klasę `active`